docs/api-reference/catalogd-webserver.md

```diff
@@ -0,0 +1,88 @@
+# Catalogd web server
+
+[Catalogd](https://github.com/operator-framework/catalogd), the OLM v1 component for making catalog contents available on cluster, includes
+a web server that serves catalog contents to clients via a HTTP(S) endpoint.
```

Comment:
This would be "an HTTP(s)" since HTTP is pronounced with a starting vowel sound. You determine the article via the acronym pronunciation.

Reply:
Should be taken care of in https://github.com/operator-framework/operator-controller/pull/1331/commits/647a9f78d0bc26b96318183daebb532b7c29dafc